.changeset/good-walls-sing.md

```diff
@@ -0,0 +1,16 @@
+---
+'@ag.ds-next/react': minor
+---
+
+`Checkbox`, `Radio` and `ControlGroup` has been moved into their own entrypoints. 
```

Comment:
```suggestion
`Checkbox`, `Radio` and `ControlGroup` have been moved into their own entrypoints. 
```

Reply:
Thanks @mikehazell, I have applied your change to this changeset in [this commit](https://github.com/steelthreads/agds-next/pull/1172/commits/45aac7c8f38069841c9fb72d7b15b1a4ff3f36e1).

I have also applied the same change to other related changesets for consistency.